src/components/bcros/filing/common/FiledLabel.vue

```diff
@@ -1,26 +1,21 @@
 <template>
-  <div class="filed-label d-inline">
-    <template v-if="isTypeStaff && putBackOnOrAdminDissolution">
-      <span>
-        {{ capitalizedFiledBy }} {{ filing.submitter }} {{ $t('text.filing.on') }}
-        <BcrosTooltipDate :date="filing.submittedDate" />
-      </span>
-    </template>
-
-    <template v-else>
-      <div class="flex flex-row gap-2 w-full">
-        <span>
-          ({{ $t('text.filing.filedBy') }} {{ filing.submitter }} {{ $t('text.filing.on') }}
-          <BcrosTooltipDate :date="filing.submittedDate" />)
-        </span>
-        <UDivider orientation="vertical" :ui="{ border: { base: 'border-gray-600'} }" />
-        <span>
-          {{ $t('text.filing.effectiveAsOf').toString() }}
-          <BcrosTooltipDate :date="filing.effectiveDate" />
-        </span>
-      </div>
-    </template>
-  </div>
+  <template v-if="isTypeStaff && putBackOnOrAdminDissolution">
+    <span>
+      {{ capitalizedFiledBy }} {{ filing.submitter }} {{ $t('text.filing.on') }}
+    </span>
+    <BcrosTooltipDate :date="filing.submittedDate" />
+  </template>
+  <template v-else>
+    <span>
+      ({{ $t('text.filing.filedBy') }} {{ filing.submitter }} {{ $t('text.filing.on') }}
+    </span>
+    <BcrosTooltipDate :date="filing.submittedDate" />)
+    <BcrosDivider />
+    <span>
+      {{ $t('text.filing.effectiveAsOf') }}
+    </span>
+    <BcrosTooltipDate :date="filing.effectiveDate" />
+  </template>
```

Comment:
Please rebase to pick up the recent corrections change, and run gcbrun again.

Here's what I see right now.
- The Court Order, Registrar's Order and Registrar's Notation should not have an effective date.
- I think "filed by" should be in parentheses, or have some other way to separate it from the status (FILED).

![image](https://github.com/user-attachments/assets/8d774476-0d93-4a78-9efc-ec6f739643a9)

Reply:
This is what I see now. Is it correct?

![image](https://github.com/user-attachments/assets/9ecc729b-398b-4e15-a132-bc0b31020113)

Reply:
Yes. I believe now all the filing items are displayed as expected.